tests/test_async_client.py

```diff
@@ -0,0 +1,91 @@
+import io
+import pathlib
+
+import pytest
+
+import saritasa_s3_tools
+
+
+@pytest.mark.usefixtures("anyio_backend")
+async def test_upload(
+    async_s3_client: saritasa_s3_tools.AsyncS3Client,
+) -> None:
```

Comment:
ANN201 should be ignored by ruff settings?
By this https://github.com/saritasa-nest/saritasa-s3-tools/pull/1/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R165-R168

Reply:
It's added via vscode automatically by double click